react/features/subtitles/middleware.ts

```diff
@@ -231,7 +247,31 @@ function _endpointMessageReceived(store: IStore, next: Function, action: AnyActi
             // after the stable part.
             newTranscriptMessage.unstable = text;
         }
+    }
+
+    if (newTranscriptMessage) {
+        if (newTranscriptMessage.final) {
+            const cachedTranscriptMessage
+                = state['features/subtitles']._cachedTranscriptMessages?.get(transcriptMessageID);
+
+            if (cachedTranscriptMessage) {
+                const cachedText = (cachedTranscriptMessage.stable || cachedTranscriptMessage.unstable)?.trim();
+                const newText = newTranscriptMessage.final;
 
+                if (cachedText && cachedText.length > 0 && newText && newText.length > 0
```

Comment:
ditto optional chaining

Reply:
Would have done that but TS complains that we are comparing undefined and number....